stable/acs-engine-autoscaler/templates/deployment.yaml

```diff
@@ -41,16 +41,28 @@ spec:
           valueFrom:
             secretKeyRef:
               name: {{ template "fullname" . }}
-              key: client-private-key   
+              key: client-private-key
         command:
             - python
             - main.py
-            - --resource-group 
+            - --resource-group
             - {{ .Values.acsenginecluster.resourcegroup }}
-            {{- if .Values.acsenginecluster.acsdeployment}}
-            - --acs-deployment 
+            {{- if .Values.acsenginecluster.acsdeployment }}
+            - --acs-deployment
             - {{ .Values.acsenginecluster.acsdeployment }}
-            {{- end}}
+            {{ end }}
+            {{- if .Values.acsenginecluster.sleeptime }}
+            - --sleep
+            - {{ .Values.acsenginecluster.sleeptime }}
+            {{ end }}
+            {{- if .Values.acsenginecluster.ignorepools }}
+            - --ignore-pools
+            - {{ .Values.acsenginecluster.ignorepools }}
+            {{ end }}
+            {{- if .Values.acsenginecluster.spareagents }}
+            - --spare-agents
+            - {{ .Values.acsenginecluster.spareagents }}
+            {{ end }}
```

Comment:
@bnookala Seeing that this PR deals alot with whitespaces, can you pls take another look at trimming left and right whitespaces. Seems you have removed few  `{{-` . Is that intended? https://github.com/kubernetes/helm/blob/master/docs/chart_template_guide/control_structures.md